x-pack/plugins/canvas/public/state/selectors/workpad.js

```diff
@@ -89,8 +89,9 @@ export function getElementCounts(state) {
 
   Object.keys(resolvedArgs).forEach(resolvedArg => {
     const arg = resolvedArgs[resolvedArg];
-    const { expressionRenderable } = arg;
-    const { value, state } = expressionRenderable;
+    const { expressionRenderable, expressionContext } = arg;
+    const expressionObject = expressionRenderable || expressionContext;
+    const { value, state } = expressionObject;
```

Comment:
This also fails if you're using `expressionContext`. Both values will be undefined, the values under that key have a numbered index (which corresponds to the index of the associated function in the expression).

![screenshot 2019-03-05 13 25 03](https://user-images.githubusercontent.com/404731/53835249-39b7fe00-3f4a-11e9-9718-97c1999d58ad.png)